src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,54 +1,101 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './FindMovie.scss';
+import classNames from 'classnames';
+import { getFilm } from '../../films';
+import { MovieCard } from '../MovieCard';
 
-// import { MovieCard } from '../MovieCard';
-
-export const FindMovie: React.FC = () => (
-  <>
-    <form className="find-movie">
-      <div className="field">
-        <label className="label" htmlFor="movie-title">
-          Movie title
-        </label>
-
-        <div className="control">
-          <input
-            type="text"
-            id="movie-title"
-            placeholder="Enter a title to search"
-            className="input is-danger"
-          />
-        </div>
+type Props = {
+  updateMovies: (movie: Movie) => void;
+};
 
-        <p className="help is-danger">
-          Can&apos;t find a movie with such a title
-        </p>
-      </div>
+export const FindMovie: React.FC<Props> = (props) => {
+  const [movieTitle, setMovieTitle] = useState<string>('');
```

Comment:
```suggestion
  const [movieTitle, setMovieTitle] = useState('');
```

type for primitive is unnecessary